tests/installation/npmTest.ts

```diff
@@ -93,12 +94,15 @@ export type NPMTestFixtures = {
 export const test = _test
     .extend<CommonFixtures, CommonWorkerFixtures>(commonFixtures)
     .extend<NPMTestFixtures>({
-      _auto: [async ({ tmpWorkspace, exec }, use) => {
+      _browsersPath: async ({tmpWorkspace}, use) => use(path.join(tmpWorkspace, 'browsers')),
+      _auto: [async ({ tmpWorkspace, exec, _browsersPath }, use) => {
         await exec('npm init -y');
         const sourceDir = path.join(__dirname, 'fixture-scripts');
         const contents = await fs.promises.readdir(sourceDir);
         await Promise.all(contents.map(f => fs.promises.copyFile(path.join(sourceDir, f), path.join(tmpWorkspace, f))));
         await use();
+        // Browsers are large, we remove them after each test to save disk space.
+        await fs.promises.rm(_browsersPath, { recursive: true, force: true });
```

Comment:
Can we remove only for successful tests? This way debugging unsuccessful tests would be easier

Reply:
done